docs/content/design/index.md

```diff
@@ -47,16 +51,14 @@ The compacted version of our original raw data looks something like this:
      2011-01-01T02:00:00Z  bieberfever.com    google.com  Male   UK      3194        170    34.01
 
 In practice, we see that rolling up data can dramatically reduce the size of data that needs to be stored (up to a factor of 100).
+Druid will roll up data as it is ingested to minimize the amount of raw data that needs to be stored. 
 This storage reduction does come at a cost; as we roll up data, we lose the ability to query individual events. Phrased another way,
-the rollup granularity is the minimum granularity you will be able to query data at. Hence, Druid ingestion specs define this granularity as the `queryGranularity` of the data.
-The lowest `queryGranularity` is millisecond.
-
-Druid is designed to perform single table operations and does not currently support joins.
-Although many production setups instrument joins at the ETL level, data must be denormalized before it is loaded into Druid.
+the rollup granularity is the minimum granularity you will be able to explore data at. Hence, Druid ingestion specs define this granularity as the `queryGranularity` of the data.
```

Comment:
Do we want to mention that events are "floored" to the query granularity? meaning that if you have hourly granularity every event will be at 00:00 of the hour?